authors/apps/authentication/tests/test_user_verification.py

```diff
@@ -0,0 +1,113 @@
+import time
+
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from authors.apps.authentication.serializers import RegistrationSerializer
+from authors.apps.authentication.views import generate_token
+
+
+class AuthenticationTests(APITestCase):
+    def setUp(self):
+        """
+        Data for the tests
+        """
+        self.data = {
+            "user": {
+                "username": "Jacob",
+                "email": "jake@jake.jake",
+                "password": "jakejake23"
+            }
+        }
+        # Set up the registration url.
+        self.reg_url = reverse('authentication:reg')
+        self.activate_url = 'authentication:activate'
+
+    def test_invalid_token_passed(self):
+        """
+        Tests if error 401 is returned if the activation URL is accessed
+        with an invalid token
+        :return: None
+        """
+        self.activate_url = reverse(self.activate_url,
+                                    kwargs={'token': "hajshja"})
+        response = self.client.get(
+            self.activate_url)
+
+        assert ("Invalid token" in str(response._container))
```

Comment:
I think it would have been better to use AssertIn instead of the assert an then pass a statement to it.